.github/workflows/publish.yml

```diff
@@ -44,30 +44,18 @@ jobs:
           need-rocksdb: true
           need-protoc: true
 
-      - name: Publish opendal
+      - name: Dryrun opendal
         working-directory: "core"
-        run: cargo publish --all-features
+        if: "contains(github.ref, '-')"
+        run: cargo publish --all-features --dry-run
         env:
           LD_LIBRARY_PATH: ${{ env.JAVA_HOME }}/lib/server:${{ env.LD_LIBRARY_PATH }}
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
 
-      - name: Publish object_store_opendal
-        working-directory: "integrations/object_store"
-        run: cargo publish
-        env:
-          LD_LIBRARY_PATH: ${{ env.JAVA_HOME }}/lib/server:${{ env.LD_LIBRARY_PATH }}
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-
-      - name: Publish oli
-        working-directory: "bin/oli"
-        run: cargo publish
-        env:
-          LD_LIBRARY_PATH: ${{ env.JAVA_HOME }}/lib/server:${{ env.LD_LIBRARY_PATH }}
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-
-      - name: Publish oay
-        working-directory: "bin/oay"
-        run: cargo publish
```

Comment:
Are these publish jobs already covered by `-all-features`?

Reply:
> Are these publish jobs already covered by `-all-features`?

No, they should not be published as we marked them as WIP.

Reply:
Right.